.github/workflows/phpunit-tests.yml

```diff
@@ -124,14 +124,15 @@ jobs:
   # - Submit the test results to the WordPress.org host test results.
   # - todo: Configure Slack notifications for failing tests.
   test-php:
-    name: ${{ matrix.php }}${{ matrix.memcached && ' with memcached' || '' }} on ${{ matrix.os }}
+    name: ${{ matrix.php }}${{ matrix.multisite && ' multisite' || '' }}${{ matrix.memcached && ' with memcached' || '' }} on ${{ matrix.os }}
     needs: setup-wordpress
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        php: [ '8.0', '7.4', '7.3', '7.2', '7.1', '7.0', '5.6.20' ]
+        php: [ '5.6.20', '8.0', '7.4', '7.3', '7.2', '7.1', '7.0' ]
```

Comment:
I think the idea here is to put the slowest job first? What about just flipping the matrix so that they stay in an order, but instead just go from oldest to newest?

Reply:
Yep I added a note to the PR description about this. It's not strictly related to this change but the theory is that if there's a job backlog then you want the slowest jobs starting first. Reversing the complete order of the list makes sense.